src/sensor/ping360helperservice.h

```diff
@@ -46,10 +62,24 @@ class Ping360EthernetFinder : public QObject {
      * @brief Stops broadcasting the discovery message
      *
      */
-    void stop();
+    void stopBroadcastService();
 
-signals:
+    /**
+     * @brief Configure device under specific IP and port as DHCP client
```

Comment:
port? is is in the IP:Port format?

Reply:
Oh, I thought that I have fixed this comments, thanks !